src/graphql/handler.js

```diff
@@ -52,6 +53,8 @@ export default startServerAndCreateLambdaHandler(
     context: async ({ event }) => {
       const { body, headers } = event
 
+      const { Host } = headers
```

Comment:
What is the host here, this feels like something we should be reading from our configs/ENV vars.

Reply:
Host here is the host where cmr-graphQL is currently running. localhost if running locally or graphql.<env.>earthdata.nasa.gov if running in AWS. This gets passed onto the CollectionVariableDrafts class so it knows where to make the earthdata-varinfo call from. Will add some comments to the code.